extras/data/machinery/generate/COMPASS.py

```diff
@@ -0,0 +1,45 @@
+from .utils import load, obs_template
+
+
+def dump(src_path, target):
```

Comment:
It looks that the `dump` function is the same for all the experiments, so you can put it top level and import it.

Reply:
I think this is because @toonhasenack  is automating even the generation of the metadata.